src/glyphs.js

```diff
@@ -57,6 +64,28 @@ const processCacheRequest = req => {
   return processed;
 };
 
+const processMessageRequest = req => {
+  const processed = {
+    errors: [],
+    parsed: {}
+  };
+
+  if (!req.body.message || !req.body.prob) {
```

Comment:
Are you okay with not abbreviating `probability`?

Reply:
Yup